postgres/datadog_checks/postgres/postgres.py

```diff
@@ -147,7 +184,7 @@ def _get_debug_tags(self):
 
     def _get_service_check_tags(self):
         service_check_tags = []
-        service_check_tags.extend(self._config.tags)
+        service_check_tags.extend(self.tags)
```

Comment:
I think we actually want to skip that one. There's nothing in the agent where we transform the internal resource tag into something useful right now. We'll have to think about what we do with the service check tagging but it's going to require another solution.